.github/workflows/ci.yml

```
@@ -10,7 +10,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-
+      
+      # Set up Python 3.11 explicitly for use in subsequent steps
+      - name: Set up Python
+        uses: actions/setup-python@v2
```

actionlint

the runner of "actions/setup-python@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)